kubera/install/experiments/assert-kubera-svc.yaml

```diff
@@ -0,0 +1,219 @@
+apiVersion: dope.mayadata.io/v1
+kind: Recipe
+metadata:
+  name: assert-kubera-svc-running
+  namespace: d-testing
+  labels:
+    d-testing.dope.mayadata.io/inference: "true"
+spec:
+  tasks:
+
+  # Check presence of alertstore service
+  - name: assert-running-of-alertstore-svc
+    assert:
+      state:
+        kind: Service
```

Comment:
Do we need to check if ClusterIP is not nil?
Do we need to check if ClusterIP is None in case of headless service?

Reply:
Above comment applies for all!

Reply:
As discussed we shall implement in next PR